.pre-commit-config.yaml

```diff
@@ -1,28 +1,38 @@
-default_stages: [pre-push]
+default_stages: [ pre-push ]
```

Comment:
Are you using any specific linting for YAML files? If so, we should apply it to other files as well.

Reply:
yes, a default yaml linting
applied to pre-commit-dev.yaml